packages/core/core/src/requests/TargetRequest.js

```diff
@@ -572,6 +595,17 @@ function parseEngines(
   }
 }
 
+// Used for backwards compatibility
+function postProcessDescriptor(descriptor: mixed): mixed {
```

Comment:
Should the minify and scopeHoist flag in package.json#targets be backwards compatible?